nipype/interfaces/fsl/tests/test_auto_Eddy.py

```diff
@@ -38,9 +38,9 @@ def test_Eddy_inputs():
     mandatory=True,
     ),
     in_topup_fieldcoef=dict(argstr='--topup=%s',
-    requires=['in_topup_movpar'],
+    requires=[u'in_topup_movpar'],
```

Comment:
4ea783c makes py2/py3 behavior equivalent. I'm not entirely sure what you're suggesting for instantiating each object... Does the current system not do that?